### PR TITLE
add support for transaction rollup contract addresses

### PR DESCRIPTION
**Pull request by zamrokk**
I just authorized "txr1" rollup addresses to the list of valid contract prefixes.
rollups on Mondaynet are protocol smart contract that acts like any user smart contract, taquito should allow to forge a Contract object from a rollup contract address

This is a minor change but it is blocking my development on a dapp for the Tezos rollups

### CODE REVIEW

File: packages/taquito-utils/src/constants.ts [hunk 1]
```diff
@@ -45,6 +46,7 @@ export const prefix = {
   [Prefix.TZ3]: new Uint8Array([6, 161, 164]),
   [Prefix.KT]: new Uint8Array([2, 90, 121]),
   [Prefix.KT1]: new Uint8Array([2, 90, 121]),
+  [Prefix.TXR1]: new Uint8Array([2, 90, 121]),
```
Comment (roxaneletourneau) [inline packages/taquito-utils/src/constants.ts:49]: I am wondering if the prefix should rather be [1, 128, 120, 031]. See [TXR1](https://gitlab.com/tezos/tezos/-/blob/master/src/proto_alpha/lib_protocol/tx_rollup_prefixes.ml#L35)

Comment (zamrokk) [reply packages/taquito-utils/src/constants.ts]: you are right, I fixed it and added other definition from rollups

Thank you  =)

File: packages/taquito-utils/src/constants.ts [hunk 2]
```diff
@@ -79,14 +91,25 @@ export const prefix = {
   [Prefix.TZ]: new Uint8Array([2, 90, 121]),
 
   [Prefix.VH]: new Uint8Array([1, 106, 242]),
+
+  [Prefix.TXR1]: new Uint8Array([1, 128, 120, 31]),
+  [Prefix.TXI]: new Uint8Array([79, 148, 196]),
+  [Prefix.TXM]: new Uint8Array([79, 149, 30]),
+  [Prefix.TXC]: new Uint8Array([79, 148, 17]),
+  [Prefix.TXMR]: new Uint8Array([18, 7, 206, 087]),
```
Comment (roxaneletourneau) [inline packages/taquito-utils/src/constants.ts:99]: Hi @zamrokk 
The build fails because there should be no '0' before '87'.

**Comment by zamrokk** [comment]
Sorry I was out for holidays

Thank you for the unit tests fix, btw

What do you need to pass the last steps ?

**Comment by roxaneletourneau** [comment]
all good, thanks @zamrokk 
It will be included in the next release shortly.

**Pull request merged**